packages/gatsby-cli/src/reporter/redux/types.ts

```diff
@@ -70,6 +70,7 @@ export interface IPendingActivity {
     id: string
     type: ActivityTypes
     status: ActivityStatuses
+    startTime?: [number, number]
```

Comment:
Shouldn't this just be `Array<number>`?

Reply:
This is to match signature of `process.hrtime([arg])` function which looks like this:

```typescript
interface HRTime {
  (time?: [number, number]): [number, number];
}
```

And additionally this was copied from already existing one in `IActivity` interface - https://github.com/gatsbyjs/gatsby/blob/318a8a2abb1518b7b011158a25939f9dc9990ba3/packages/gatsby-cli/src/reporter/redux/types.ts#L24

Reply:
:shrug: ok